src/invidious/jsonify/api_v1/video_json.cr

```diff
@@ -245,6 +245,8 @@ module Invidious::JSONify::APIv1
                 json.field "lengthSeconds", rv["length_seconds"]?.try &.to_i
                 json.field "viewCountText", rv["short_view_count"]?
                 json.field "viewCount", rv["view_count"]?.try &.empty? ? nil : rv["view_count"].to_i64
+                json.field "published", rv["published"]?
+                json.field "publishedTimeText", translate(locale, "`x` ago", rv["publishedText"].to_s.gsub(" ago", ""))
```

Comment:
Two things here:
1. The JSON field should be `publishedText`, to match the rest of the API
2. The text should be translated from the time we parsed earlier. The text from the innertube (internal Youtube) API is always in english, and can't be used directly.

Reply:
I had trouble fully understanding the second request, but I improved my old code. I thought about using `Time.unix(0)` when `rv[published]` was `nil`, but decided to make it more obvious that something is wrong, when it happens.

I also greatly apologize for the 3 month delay! I took a very long break, because of personal issues in my life.

Reply:
> I also greatly apologize for the 3 month delay! I took a very long break, because of personal issues in my life.

No problem :) Your life is much more important than anything else!

Reply:
Thank you so much for saying that :)